### ISSUE TITLE
Charts not working with window smaller than 800px wide with metabase 0.34.0

### ISSUE THREAD

**Issue by pulkitjalan** [metabase/metabase#11619]
**Describe the bug**
Line graphs and bar charts are not working properly on mobile.

**Logs**
![Screenshot 2020-01-02 at 13 26 37](https://user-images.githubusercontent.com/4124930/71669967-5be6fc00-2d66-11ea-910e-f56c68ae8880.png)

**To Reproduce**
1. Create a new line graph
2. Reduce the screen size to less than 800px, or view on mobile

**Expected behavior**
They are working fine on older versions.

**Screenshots**
This is what it looks like on mobile:
![Screenshot 2020-01-02 at 13 46 39](https://user-images.githubusercontent.com/4124930/71670036-9781c600-2d66-11ea-81e0-97b91576acd2.png)
All the plots are supposed to be different dates.

**Information about your Metabase Installation:**

- Your browser and the version: Chrome 79.0.3945.88
- Your operating system: OS X 10.15.2
- Your databases: MySQL
- Metabase version: 0.34.0
- Metabase hosting environment: Elastic Beanstalk
- Metabase internal database: Postgres

**Severity**
Blocking users from viewing charts and dashboards on mobile.

**Comment by pulkitjalan** [comment]
Is it possible to downgrade from 0.34.0 to 0.33.7.3?

**Comment by flamber** [comment]
Hi @pulkitjalan 
I cannot reproduce your issue.
Are you seeing this problem with other browsers than Chrome? Have you tried without extensions?
Can you reproduce the problem by creating a question from Sample Dataset?

Yes, you can downgrade between those two versions. Simply follow the same guide as upgrading, but just change the download link to 0.33.7.3: https://www.metabase.com/docs/latest/operations-guide/running-metabase-on-elastic-beanstalk.html#deploying-new-versions-of-metabase
Always make a backup before downgrading/upgrading.

**Comment by pulkitjalan** [comment]
Hi @flamber, thanks for the quick response!

I have been able to reproduce the issue with different browsers too. Safari, Firefox & Chrome on OS X. Safari & Chrome on iOS. I also tried without extensions and was still able to reproduce it.

I will try thee sample dataset on a local setup and get back to you.

I did a bit more testing and it seems to be only with our graphs that have a 30 or 31 days filter. If I increase that filter to anything past 32 days it works fone.

**Comment by flamber** [comment]
@pulkitjalan Perfect. I've reproduced it now :+1: 

**Reproduce**
1. Simple question > Sample Data > Orders
2. Filter CreatedAt previous 30 days
3. Summarize Count-of-rows by CreatedAt:Day
4. Resize browser window to less than 800px, will not show graph and produces a lot of console logs

**Logs**
```
Firefox:
Unexpected value translate(NaN,0) parsing transform attribute.
Unexpected value NaN parsing x attribute.
Unexpected value NaN parsing width attribute.
Unexpected value translate(NaN,0) parsing transform attribute.
Unexpected value NaN parsing x attribute.
Unexpected value NaN parsing width attribute.

Chromium:
Error: <path> attribute d: Expected number, "MNaN,6V0HNaNV6".
Error: <g> attribute transform: Expected number, "translate(NaN,0)".
Error: <g> attribute transform: Expected number, "translate(NaN,0)".
Error: <rect> attribute x: Expected length, "NaN".
Error: <rect> attribute width: Expected length, "NaN".
Error: <path> attribute d: Expected number, "MNaN,6V0HNaNV6".
Error: <g> attribute transform: Expected number, "translate(NaN,0)".
Error: <g> attribute transform: Expected number, "translate(NaN,0)".
Error: <rect> attribute x: Expected length, "NaN".
Error: <rect> attribute width: Expected length, "NaN".
```

**Comment by pulkitjalan** [comment]
Hi @flamber,

I was able to reproduce it on a fresh install with the sample dataset.

<img width="1680" alt="Screenshot 2020-01-02 at 15 39 33" src="https://user-images.githubusercontent.com/4124930/71675693-3e219300-2d76-11ea-9f9a-5c721536308d.png">
<img width="797" alt="Screenshot 2020-01-02 at 15 39 56" src="https://user-images.githubusercontent.com/4124930/71675695-3e219300-2d76-11ea-9df7-af04d107baef.png">

**Comment by pulkitjalan** [comment]
Ah perfect, thanks @flamber!

**Comment by ibushong** [comment]
I'm experiencing a similar issue after upgrading to v.0.34.0. All the charts on my dashboard that are supposed to show daily-aggregated stats over the past 30 days now just show one bar on Jan 1, 2020. Seems like possibly a year-transition issue?

I reverted back to v0.33.7 but now all these charts/questions are broken with this error:

```Input to store-field! does not match schema: [0;33m [(named {:base_type (not ("Valid field type" a-clojure.lang.Keyword))} field)] [0m```

In the JS console, I'm seeing a lot of `Invalid type: type/DateTimeWithLocalTZ`

**Comment by flamber** [comment]
@ibushong No, this isn't related to the year change. If you only get data between 2019-11-01 and 2019-11-30, then it fails to show the chart too.

Did you create/modify the questions on 0.34.0? If yes, then you need to re-make them on previous versions, since 0.34.0 add a lot of timezone changes. You should revert to your backup, if you have one. And use 0.33.7.3, not 0.33.7

**Comment by ibushong** [comment]
Ah ok that makes sense.

No, I did not touch these questions after updating. I will try v0.33.7.3, and also will try re-creating the questions just in case. Thanks for the help!

**Comment by Leoli0320** [comment]
hi, 
I have the same question too.
Before I upgrade to version 0.34.0, I didn't modify my question.
After that, my question in the dashboard shows like @pulkitjalan

**Comment by paulrosenzweig** [comment]
Fixed in #11646
